<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPassword</class>
 <widget class="QMainWindow" name="ForgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>忘记密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/LW_10.png</normaloff>:/res/LW_10.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background:\url(:/res/LW_13.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1151</width>
      <height>891</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background:rgba(255,166,68,0.5);
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:50px;
font-family:Microsoft JhengHei;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string>忘记密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>431</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #002D40;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>470</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>请输入密保答案</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>520</y>
       <width>431</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #002D40;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Answer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>640</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>请输入新密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>690</y>
       <width>431</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #002D40;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>New Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>640</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#96CA2D;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#168039;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>640</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#ff8c69;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
border-radius:60px;
background:#FF6F69;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>921</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
border:1px solid #002D40;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入用户名，并点击查询按钮查看您的密保问题</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:40px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>您的密保问题是</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#FFA644;
border-radius:15px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#E64661;
border-radius:15px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>110</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#002D40;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>提示：管理员用户修改密码需要携带二代身份证到图书馆二楼现场办理</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
